src/components/handle/assets/handle/t9n/messages.json

```diff
@@ -0,0 +1,3 @@
+{
+  "title": "handle"
```

Comment:
wondering if `dragHandle` makes more sense here. Thoughts @jcfranco , @geospatialem ?

Reply:
Agreed and now that you bring it up, using it as the value may also be more helpful. I'll defer to @geospatialem on that one.

Reply:
Is the context for when users are moving items? And if so, would it override what we have in place for [`value-list`](https://github.com/Esri/calcite-components/blob/master/src/components/value-list/assets/value-list/t9n/messages.json#L1-L6)? 

If its separate from `value-list`, and doesn't override those strings, dig the `dragHandle` suggestion, perhaps could also update the value? 

WDYT about:

> "dragHandle": "Drag handle"

Reply:
This is separate from `value-list` in terms of implementation  which doesn't override the instruction we have in place for `value-list`.  Since you bought it up, i wonder if `sortable-list` and block should have the same a11y support for dragEnabled as value-list 🤔 ?

Reply:
> WDYT about:
> 
> > "dragHandle": "Drag handle"

I prefer using `dragHandle`  both for value and key and since it is internal component wont be a breaking change and dragHandle is more self explanatory  than handle .

Reply:
> I wonder if `sortable-list` and `block` should have the same a11y support for `dragEnabled` as `value-list` considering the functionality is similar 🤔

Oh, good call. Yeah, would imagine we should add for inclusion as well.